framework/tlvf/tlvf.py

```diff
@@ -209,6 +209,7 @@ def __init__(self, fname, name, dict):
         self.list_index = 0
         self.lock_allocation_member_added = False
         self.lock_order_member_added = False
+        self.marker_post_init_method_added = False
```

Comment:
Can we call it `post_init_method_added` instead and set it to True after we encounter the first class member and really add the marker and the function?